dds/idl/v8_generator.cpp

```diff
@@ -122,6 +124,10 @@ namespace {
       } else if (v8Type == "Number") {
         prefix = "static_cast<double>(";
         suffix = ")";
+      } else if (v8Type == "String" && (pt == AST_PredefinedType::PT_longlong || pt == AST_PredefinedType::PT_ulonglong)) {
```

Comment:
Is the check for v8Type here redundant?  Isn't it enough to check pt?

Reply:
Hmm, yeah, redundant. I just wanted the change applied as narrowly as possible and saw checks being made against both type variables above and so I just added both. I'll fix it.